api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimRepositoryTest.java

```diff
@@ -244,4 +244,14 @@ void shouldReturnNewClaimIds() {
         //Then
         assertThat(result).containsOnly(newClaim.getId());
     }
+
+    @Test
+    void shouldReturnLazyLoadedReferenceToClaim() {
+        UUID id = UUID.randomUUID();
```

Comment:
I'm testing here that the claim has not been loaded from the DB, as doing so would throw an error. Open to better ways of testing this.

Reply:
I'm not sure I understand this test, surely it shouldn't return a result at all as the db is empty?

Reply:
The point is that it's not going to the DB, instead creating a proxy for us with the given ID. If we were to try and access any other properties I expect we'd get an error as it would then try to call the DB to populate the Claim. 
Maybe I should just remove this test - I put it here largely to reassure myself that Spring was wiring everything up correctly.

Reply:
Wow, I didn't realise that's what it was trying to achieve. Yes, this either needs re-writing so that is clear, or we just remove it.

Reply:
Perhaps a comment above the test explaining it's purpose might help.